superset-frontend/src/dashboard/components/nativeFilters/types.ts

```diff
@@ -66,3 +87,51 @@ export type FilterState = {
    */
   isDirty: boolean;
 };
+
+/** Chart state of redux */
+export type Chart = {
+  id: number;
+  slice_id: 2107;
+  formData: {
+    viz_type: string;
+  };
+};
+
+/** Root state of redux */
+export type RootState = {
```

Comment:
It would probably be better for these types to live where they're used, since they're not really specific to filters.

Reply:
👍 Moving